frontend/src/components/navbar.css

```diff
@@ -24,23 +24,30 @@
   margin: 0rem;
 }
 
+.orange {
+  color: #f19b3b;
+}
+
+.pink {
+  block-size: inline;
+  margin: right;
+  color: #f05b75;
```

Comment:
What I meant was something like:

```
  display: inline-block;
  margin-right: 0.5em;
```

The inline-block changes the display mode to an inline block which allows blocks to be inline with other text, but also have things like padding and margin that aren't available to regular text.

But if you were going to add these properties, then I would also change the class name to something that's not tied to a color since that greatly expands what this does.